image/image.go

```diff
@@ -307,6 +313,78 @@ func GetUncompressedTarContentSize(path string) (size int64, err error) {
 	}
 }
 
+// Copy copies one file or device to another.
+func Copy(dst, src string, createDst bool) (copied int64, err error) {
+	log.WithFields(log.Fields{"src": src, "dst": dst}).Debug("Copy")
+
+	srcFile, err := os.OpenFile(src, os.O_RDONLY, 0)
+	if err != nil {
+		return 0, aoserrors.Wrap(err)
+	}
+	defer srcFile.Close()
+
+	dstFlags := os.O_RDWR
+
+	if createDst {
```

Comment:
in my opinion, it would be better if the `Copy` function handles itself if needed to create a destination. I think it will improve the usability function `Copy`. What do you think?

Reply:
It is done on purpose. Thera two use cases of this function:
- copy from file to file: in this case Copy function should create dst if not exist;
- copy from file to partition: in this case dst should exist, otherwise (wrong dst path for example) copy will create file on FS and copy there. We don't have any error in this case. That may lead to undefined behavior.

Reply:
in my opinion, I spend some time when reading code on the call side because I didn't understand what it means false or true